### PR TITLE
feat: (Core) add theme switcher API service

### PR DESCRIPTION
**Pull request by katekozlowska**
#### Please provide a link to the associated issue.

fixes https://github.com/SAP/fundamental-ngx/issues/3465

#### Please provide a brief summary of this pull request.

A theme switcher API (service).

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/documentation/core-helpers/toolbar/toolbar.component.ts [hunk 1]
```diff
@@ -33,48 +36,34 @@ export class ToolbarDocsComponent implements OnInit {
...
-        private sanitizer: DomSanitizer
+        private _routerService: Router,
+        private _themesService: ThemesService,
+        private _docsThemeService: DocsThemeService,
+        @Inject('CURRENT_LIB') private _currentLib: Libraries,
     ) {
-        this.library = routerService.routerState.snapshot.url.includes('core') ? 'Core' : 'Platform';
+        this.library = _routerService.routerState.snapshot.url.includes('core') ? 'Core' : 'Platform';
+
+        this._docsThemeService.onThemeChange.subscribe(theme => {
```
Comment (Betrozov) [inline apps/docs/src/app/documentation/core-helpers/toolbar/toolbar.component.ts:60]: should we unsubscribe here?

**Comment by InnaAtanasova** [comment]
The code looks ok, the requirement specified in the ticket "URL parameter should be configurable" is present.

File: apps/docs/src/app/documentation/core-helpers/toolbar/toolbar.component.ts (hunk 1 above)
Comment (JKMarkowski) [reply apps/docs/src/app/documentation/core-helpers/toolbar/toolbar.component.ts]: Done. Thanks for catching it

**Review by stefanoScalzo — APPROVED** [review]
LGTM, just fix the subscription

**Review by droshev — APPROVED** [review]

**Pull request merged**